srcpkgs/minbif/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'minbif'
+pkgname=minbif
+version=1.0.5
+revision=1
+wrksrc="$pkgname-$version"
+build_style=cmake
+configure_args="PREFIX=/usr CONF_PREFIX=/etc/minbif ENABLE_CACA=$(vopt_if caca 'ON' 'OFF') ENABLE_VIDEO=$(vopt_if caca 'ON' 'OFF') ENABLE_PAM=ON ENABLE_TLS=ON DEBUG=OFF"
+hostmakedepends="pkg-config git"
+makedepends="libpurple-devel imlib2-devel gnutls-devel pam-devel $(vopt_if caca libcaca-devel)"
+short_desc="IRC gateway to IM networks that uses libpurple"
+maintainer="TROUVERIE Joachim <joachim.trouverie@linoame.fr>"
+license="GPL-2.0-only"
+homepage="https://symlink.me/projects/minbif/wiki/"
+
+build_options="caca"
+desc_option_caca="Build with libcaca support to display icons"
+
+do_configure() {
+	make $configure_args
+}
+
+do_fetch() {
+	git clone https://symlink.me/romain/$pkgname.git $wrksrc
```

Comment:
we **need** a versioned tarball

Reply:
Unfortunatelly I did not found any distfile for this package, I'll try to ask on the minbif IRC channel.